po/vi.po

```diff
@@ -3345,85 +3345,106 @@ msgstr ""
 
 #: src/control-flow-basics/blocks-and-scopes.md
 msgid "Blocks"
-msgstr ""
+msgstr "Blocks (khối lệnh)"
 
 #: src/control-flow-basics/blocks-and-scopes.md
 msgid ""
 "A block in Rust contains a sequence of expressions, enclosed by braces `{}`. "
 "Each block has a value and a type, which are those of the last expression of "
 "the block:"
 msgstr ""
+"Mỗi block trong Rust chứa một chuỗi những biểu thức, được bao bọc bởi dấu `{}"
+"`. Mỗi block có một giá trị và một kiểu dữ liệu, và là biểu thức cuối cùng "
+"của block đó:"
 
 #: src/control-flow-basics/blocks-and-scopes.md
 msgid "\"y: {y}\""
-msgstr ""
+msgstr "\"y: {y}\""
 
 #: src/control-flow-basics/blocks-and-scopes.md
 msgid ""
 "If the last expression ends with `;`, then the resulting value and type is "
 "`()`."
 msgstr ""
+"Nếu biểu thức cuối cùng kết thúc bằng dấu `;`, giá trị trả về và kiểu dữ "
+"liệu sẽ là `()`."
 
 #: src/control-flow-basics/blocks-and-scopes.md
 msgid ""
 "You can show how the value of the block changes by changing the last line in "
 "the block. For instance, adding/removing a semicolon or using a `return`."
 msgstr ""
+"Giáo viên có thể trình bày rằng giá trị của block thay đổi bằng cách thay "
+"đổi dòng cuối cùng của block đó. Ví dụ, thêm vào / loại bỏ dấu `;` hoặc sử "
+"dụng lệnh `return`"
 
 #: src/control-flow-basics/blocks-and-scopes/scopes.md
 msgid "A variable's scope is limited to the enclosing block."
 msgstr ""
+"Mỗi phạm vi của một biến đều được giới hạn trong một block (khối lệnh)."
 
 #: src/control-flow-basics/blocks-and-scopes/scopes.md
 msgid ""
 "You can shadow variables, both those from outer scopes and variables from "
 "the same scope:"
 msgstr ""
+"Học viên có thể shadow (che khuất) các biến,  bao gồm các biến ở ngoài phạm "
+"vi và các biến ở trong cùng phạm vi:"
 
 #: src/control-flow-basics/blocks-and-scopes/scopes.md
 msgid "\"before: {a}\""
-msgstr ""
+msgstr "\"ban đầu: {a}\""
 
 #: src/control-flow-basics/blocks-and-scopes/scopes.md src/generics/exercise.md
 #: src/generics/solution.md src/std-traits/from-and-into.md
 #: src/lifetimes/solution.md
 msgid "\"hello\""
-msgstr ""
+msgstr "\"hello\""
 
 #: src/control-flow-basics/blocks-and-scopes/scopes.md
 msgid "\"inner scope: {a}\""
-msgstr ""
+msgstr "\"phạm vi bên trong: {a}\""
```

Comment:
I don't know about this. What do you think @daivinhtran

Reply:
Hmm I'm not sure either. Can we keep it as inner scope?

Reply:
@daivinhtran I think we can still translate it to Vietnamese but put the English acronym in the brackets

Reply:
I will put the translation in the brackets like `inner scope (phạm vi bên trong)`. We should translate as much as possible because our audience might know little to no English. It's not just hard to read but also hard to pronounce thus the translation so some one can refer to if they having a hard time reading it.

Reply:
@HuyNVuong  I agree with you. Either ```phạm vi bên trong (inner scope)``` or ```inner scope (phạm vi bên trong)``` is good I believe.

Reply:
Done